source/features/first-published-tag-for-merged-pr.tsx

```diff
@@ -33,15 +33,16 @@ async function init(): Promise<void> {
 		discussionHeader.classList.add('rgh-first-tag');
 
 		discussionHeader.parentElement!.append(
-			' • ',
```

Comment:
Why remove this?

Reply:
If you look closer, you will notice GitHub uses ` • ` as a plain text separator.

In this case, the icon itself is already a strong manifestation of division. There's no need to put the dot separator here.

Also notice how GitHub arrange the entire  `byLine` as a single sentence. There's no huge gaps between. This is extremely obvious when you consider a small area of the line: _insert screenshot here_*

\*: TODO since I'm on mobile

Reply:
I think it's fine to break small GH conventions when it improves readability. Just my 2¢

Reply:
I had an alternative design: 8px gap in the header but stay 4px in the sticky header. How's that sound to you?

Reply:
I don't think I'd even notice a 4px difference honestly :smile: